test/src/androidTest/kotlin/io/realm/shared/notifications/RealmListNotificationsTests.kt

```diff
@@ -33,38 +47,207 @@ class RealmListNotificationsTests : NotificationTests {
     }
 
     @Test
-    @Ignore
     override fun initialElement() {
-        TODO("Waiting for RealmList support")
+        val dataSet = OBJECT_VALUES
+
+        realm.writeBlocking {
+            val managedContainer = copyToRealm(RealmListContainer())
+            managedContainer.objectListField
+                .addAll(dataSet)
+        }
+
+        runBlocking {
+            val channel = Channel<RealmList<*>>(capacity = 1)
+            val observer = async {
+                val container = realm.objects<RealmListContainer>()
+                    .first()
+
+                container.objectListField
+                    .observe()
+                    .collect { flowList ->
+                        channel.send(flowList)
+                    }
+            }
+
+            // Assertion after empty list is emitted
+            val emittedList = channel.receive()
+            assertNotNull(emittedList)
+            assertEquals(dataSet.size, emittedList.size)
+
+            observer.cancel()
+            channel.close()
+        }
     }
 
     @Test
-    @Ignore
     override fun observe() {
-        TODO("Waiting for RealmList support")
+        val dataSet = OBJECT_VALUES
+
+        realm.writeBlocking {
+            // Just create an empty container with empty lists
+            copyToRealm(RealmListContainer())
+        }
+
+        runBlocking {
+            val channel = Channel<RealmList<*>>(capacity = 1)
+            val observer = async {
+                val container = realm.objects<RealmListContainer>()
```

Comment:
It seems safer to use the result from the `writeBlocking` block?

```
val container = realm.writeBlocking {
            // Just create an empty container with empty lists
            copyToRealm(RealmListContainer())
        }

        runBlocking {
            val channel = Channel<RealmList<*>>(capacity = 1)
            val observer = async {
                container.objectListField.
            }
```

Here and elsewhere